testing/e2e/config/config.go

```diff
@@ -83,13 +84,14 @@ type NodeSettings struct {
 	ExecutionSettings ExecutionSettings `json:"execution_settings"`
 }
 
-// ExecutionSettings holds the configuration for the execution layer
-// clients.
+// ExecutionSettings holds the configuration for the execution layer clients.
 type ExecutionSettings struct {
 	// Specs holds the node specs for all nodes in the execution layer.
 	Specs NodeSpecs `json:"specs"`
 	// Images specifies the images available for the execution layer.
 	Images map[string]string `json:"images"`
+	// NethermindConfig holds specific configuration for Nethermind client
+	NethermindConfig *NethermindConfig `json:"nethermind_config,omitempty"`
 }
```

Comment:
The e2e tests are currently failing, build is failing due to errors. Pls have a look. 
You can try running `make test-e2e` on your machine, I am moving this PR to draft, once it is ready let us know.